### ISSUE TITLE
NTR: Cardiac imaging: right ventricle aneurysm (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8999]
New term request Cardiac imaging: right ventricle aneurysm (elthat_140404183519): Echocardiogram or MRI; cardiac ultrasound; heart; dilation; dilatation  UMLS: 2040231

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000667
- label: Right ventricular aneurysm
- parents: [Abnormal right ventricle morphology (HP:0001707)](https://hpo.jax.org/app/browse/term/HP:0001707)
- pmid: PMID:34256889
- def: A localized, abnormal dilation or bulging of a portion of the right ventricular wall.
- comment: Right ventricular aneurysm can be observed following myocardial infarction, trauma, or infections and with certain congenital heart defects.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8999
- synonyms: Right ventricular outpouching(NONE)

**Comment by MickeySegal** [comment]
I see we also have https://hpo.jax.org/app/browse/term/HP:0005133.  Should we retain both?

**Comment by pnrobinson** [comment]
I updating the definition and adding this comment:

Ventricular dilatation is to be distinguished from ventricular aneurysm, which refers to a diastolic as well as systolic out-pouching of the ventricle containing endocardium, epicardium, and thin-wall non-contractile scarred myocardium, which mainly develop in the setting of a full-thickness infarct, replaced by fibrous tissue because of myocardial infarction.
